From: popcornmix <popcornmix@gmail.com>
Date: Fri, 17 Apr 2020 16:24:21 +0100
Subject: configs: Enable CONFIG_ARM_BCM2835_CPUFREQ

We actually use the 64-bit 2711 kernel on Pi3 too.
It still uses the downstream cpufreq driver in DT.

See: https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=269769&start=100#p1644752
Signed-off-by: popcornmix <popcornmix@gmail.com>
---
 arch/arm64/configs/bcm2711_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index dac95c734828..14398af7f558 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -55,7 +55,6 @@ CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_CPU_FREQ_GOV_SCHEDUTIL=y
 CONFIG_CPUFREQ_DT=y
 CONFIG_ARM_RASPBERRYPI_CPUFREQ=y
-# CONFIG_ARM_BCM2835_CPUFREQ is not set
 CONFIG_RASPBERRYPI_FIRMWARE=y
 CONFIG_VIRTUALIZATION=y
 CONFIG_KVM=y
